nginx-proxy/nginx.conf

```diff
@@ -15,9 +15,29 @@ http {
     default_type application/octet-stream;
     charset_types text/css text/plain text/vnd.wap.wml application/javascript application/json application/rss+xml application/xml;
 
-    log_format  main  '$remote_addr - $remote_user [$time_local] "$request" '
-                      '$status $body_bytes_sent "$http_referer" '
-                      '"$http_user_agent" "$http_x_forwarded_for"';
+    log_format main escape=json '{'
```

Comment:
Does it work if `json_log` is referenced after that particular config is loaded [here](https://github.com/futurelearn/dockerfiles/pull/9/files#diff-b1dc99521b923cc30faedd827b5fbcbdL47)? 

This may avoid defining it twice.